projects/packages/forms/src/dashboard/inbox/response.js

```diff
@@ -12,6 +15,10 @@ const InboxResponse = () => {
 		'is-name': true,
 	} );
 
+	if ( ! loading && ! response ) {
```

Comment:
Nit: I think we just leave the column empty if there's nothing to show. So we could just `return null;` here and be done with it?